packages/grafana-ui/src/components/Drawer/Drawer.tsx

```diff
@@ -112,35 +112,21 @@ export function Drawer({
           ref={overlayRef}
         >
           {typeof title === 'string' && (
-            <div className={styles.header}>
+            <div className={cx(styles.header, Boolean(tabs) && styles.headerWithTabs)}>
               <div className={styles.actions}>
-                {expandable && !isExpanded && (
-                  <IconButton
-                    name="angle-left"
-                    size="xl"
-                    onClick={() => setIsExpanded(true)}
-                    aria-label={selectors.components.Drawer.General.expand}
-                  />
-                )}
-                {expandable && isExpanded && (
-                  <IconButton
-                    name="angle-right"
-                    size="xl"
-                    onClick={() => setIsExpanded(false)}
-                    aria-label={selectors.components.Drawer.General.contract}
-                  />
-                )}
-                <IconButton
-                  name="times"
-                  size="xl"
+                <Button
+                  icon="times"
+                  variant="secondary"
+                  fill="text"
                   onClick={onClose}
                   aria-label={selectors.components.Drawer.General.close}
                 />
               </div>
               <div className={styles.titleWrapper}>
-                <h3 {...titleProps}>{title}</h3>
-                {typeof subtitle === 'string' && <div className="muted">{subtitle}</div>}
-                {typeof subtitle !== 'string' && subtitle}
+                <Text as="h3" {...titleProps}>
```

Comment:
🎉 

You can alternatively just import `H3` and use `<H3 {...titleProps}>` , which one do you prefer? We are open to feedback for the best use of this.

Reply:
@JoaoSilvaGrafana did not know that :) think I prefer Text for this use case